src/test/java/racinggame/domain/CarsTest.java

```diff
@@ -0,0 +1,79 @@
+package racinggame.domain;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Iterator;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.junit.jupiter.api.Test;
+import racinggame.domain.Car;
+import racinggame.domain.Cars;
+import racinggame.random.Radom;
+
+public class CarsTest {
+
+    @Test
+    public void 자동차_갯수_카운트테스트(){
+        List<Car> list =carList(0,1,2,3,4);
+        Cars cars = new Cars(list);
+        assertThat(cars.count()).isEqualTo(list.size());
+    }
+
+    @Test
+    public void 자동차_생성테스트() {
+        List<Car> list = carList(0,1,2,3,4);
+        Cars cars = new Cars(list);
+        assertThat(cars).isEqualTo(new Cars(list));
+        assertThat(cars.count()).isEqualTo(list.size());
+    }
```

Comment:
> 자바 코드 컨벤션을 지키면서 프로그래밍한다.
이 과정의 Code Style은 [intellij idea Code Style. Java](https://www.jetbrains.com/help/idea/code-style-java.html)을 따른다.
[intellij idea Code Style. Java](https://www.jetbrains.com/help/idea/code-style-java.html)을 따르려면 code formatting 단축키(Windows : Ctrl + Alt + L. Mac : ⌥ (Option) + ⌘ (Command) + L.)를 사용한다.

```suggestion
    @Test
    public void 자동차_갯수_카운트테스트(){
        List<Car> list = carList(0, 1, 2, 3, 4);
        Cars cars = new Cars(list);
        assertThat(cars.count()).isEqualTo(list.size());
    }

    @Test
    public void 자동차_생성테스트() {
        List<Car> list = carList(0, 1, 2, 3, 4);
        Cars cars = new Cars(list);
        assertThat(cars).isEqualTo(new Cars(list));
    }
```

isEqualTo 검증으로 카운트는 중복 검증일 수 있어보이네요!